### DISCUSSION TITLE
Custom Easing Handle Bezier

### DISCUSSION THREAD

**Discussion by yudopr** [keiwando/toonsquid#85] [Ideas]
Hi, this is an amazing app.
I come from Adobe Flash back then and moved to After Effects since the SWF died. This App is as close as it gets for motion graphics and I am trying as best as I can to stay away from Adobe products with its subscription model. All and all, thank you for single-handedly creating this app.

I don't know if this is under the correct category but it would be nice to have a custom easing handle just like the path bezier handle that we can set as we please. You know something like this.
<img width="223" alt="Flow Plugin for After Effects" src="https://user-images.githubusercontent.com/3666980/205297676-fb5cdf17-7511-47cc-8359-ee4adf516959.png">
Here's the plugin page Flow for After Effects https://aescripts.com/flow/

Again, I thank you and your talent is such a blessing to the community 😄
Looking forward to your answer.

**Comment by ArtistBirdy** [comment]
Yes this is similar to the spline editor in davinchi resolve's fusion where you can edit keyframe Bezier direct in time line without presets.

![image](https://user-images.githubusercontent.com/118920457/205416951-ad40f20c-b2d9-4838-ba6a-590ef28f72d4.jpeg)

**Comment by keiwando** [comment]
Thank you very much, you are very kind! :)
Yes, I have plans to add custom bezier easing curves and a graph editor in the future.

**Reply by yudopr** [reply]
Oh, and a precise scroller like a trackpad that can change the value of a property in a swipe would be nice too 😅
Sorry if I ask too much. Again, thank you very much.
